### DISCUSSION TITLE
How translate response error in return create User

### DISCUSSION THREAD

**Discussion by denison** [Laravel-Backpack/community-forum#584] [General]
Hi guys, I'm new to backpack.

How translate reponse message the request validation ?
![WhatsApp Image 2023-07-24 at 17 29 06](https://github.com/Laravel-Backpack/community-forum/assets/6712969/bc6af6f8-3652-4b53-9a3a-b70c934b5f99)

I m from Brazil, and I want to translate everything to Portuguese, but these answers I'm not getting :(

Thanks..

**Comment by jcastroa87** [comment]
hello @denison first you can try to set "locale" in

`config/app.php`

or you can do this using FormRequest: https://backpackforlaravel.com/docs/6.x/crud-operation-create#a-validating-fields-using-formrequests and add "messages", FormRequest are locate in

`app/Http/Requests`

Cheers.

**Reply by denison** [reply]
Perfect ... Thanks men.

**Reply by tabacitu** [reply]
I would only create custom messages for "custom stuff". Backpack itself doesn't provide any custom validation messages. All validation messages are provided by Laravel, but the problem is... Laravel only provides them in English 😉 There's a super-good package that provides translations for Laravel strings for other languages - https://github.com/Laravel-Lang/lang - I heavily recommend you use it if you have a non-English website.

I think MOST validation strings will be translated by installing that.
